org.eclipse.xtext.testlanguages/build.gradle

```diff
@@ -4,9 +4,7 @@ dependencies {
 	compile project(':org.eclipse.xtext')
 	compile project(':org.eclipse.xtext.testing')
 	compile "org.eclipse.platform:org.eclipse.equinox.common:$versions.equinoxCommon"
-	compile("org.eclipse.platform:org.eclipse.core.runtime:$versions.eclipseCore") {
-		exclude group: "org.eclipse.platform", module: "org.eclipse.equinox.common"
-	}
+	compile "org.eclipse.platform:org.eclipse.core.runtime:$versions.eclipseCore"
```

Comment:
No karsten this is still needed. Only the bootstrap thing should go

Reply:
OK, will update

Reply:
That affects all but bootstrap

Reply:
still needs change